### ISSUE TITLE
Missing Prerequisites: Microsoft.EntityFrameworkCore.Tools

### ISSUE THREAD

**Issue by VandeurzenDries** [dotnet/EntityFramework.Docs#3927]
[Enter feedback here]
The page mentions under prerequisites that 'nstall-Package Microsoft.EntityFrameworkCore.Design' is needed.
After installing this (via pmc ) and continuing to the  'Scaffold-DbContext ...' step you will get an error.
This error is solved by installing 'Install-Package Microsoft.EntityFrameworkCore.Tools'  (via pmc). 

A mention of the 'tools' package in the prerequisites might be helpfull so that other users don't encounter this issue.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e1601c3e-3f29-5d84-6350-2b38eeaf6adc
* Version Independent ID: 92a856d6-c5da-e215-46ac-d67c411f3ff9
* Content: [Reverse Engineering - EF Core](https://docs.microsoft.com/en-us/ef/core/managing-schemas/scaffolding?tabs=dotnet-core-cli)
* Content Source: [entity-framework/core/managing-schemas/scaffolding.md](https://github.com/dotnet/EntityFramework.Docs/blob/main/entity-framework/core/managing-schemas/scaffolding.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @bricelam
* Microsoft Alias: **avickers**

**Comment by bricelam** [comment]
Indeed. We should change Microsoft.EntityFrameworkCore.Design to Tools in the prerequisites since this will work for both the .NET Core CLI and Visual Studio.
